### PR TITLE
Add Prometheus version check for faas

### PR DESCRIPTION
**Pull request by ebondare**

### CODE REVIEW

**Comment by ebondare** [comment]
Successful test run: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/25115/console

File: ocs_ci/helpers/managed_services.py [hunk 1]
```diff
@@ -525,6 +526,14 @@ def verify_faas_provider_resources():
     log.info("Verifying managedFusionOffering onboardingValidationKey")
     assert len(offering_info["spec"]["config"]["onboardingValidationKey"]) > 700
 
+    # Verify that version of Prometheus is 4.10.0
+    prometheus_csv = csv.get_csvs_start_with_prefix(
+        constants.OSE_PROMETHEUS_OPERATOR, config.ENV_DATA["cluster_namespace"]
+    )
+    prometheus_version = prometheus_csv[0]["spec"]["version"]
+    log.info(f"Verifying Prometheus version: {prometheus_version}")
+    assert prometheus_version == "4.10.0"
```
Comment (fbalak) [inline ocs_ci/helpers/managed_services.py:535]: Wouldn't it be better to take the version from config file? It can be in defaults

Comment (ebondare) [reply ocs_ci/helpers/managed_services.py]: Thank you! Moved the version to the default config file

File: ocs_ci/helpers/managed_services.py [hunk 2]
```diff
@@ -418,6 +418,7 @@ def verify_faas_provider_resources():
     7. Check the presence of subscription and its health
     8. Check that mon PVCs have gp3-csi storageclass
     9. Check managedFusionOffering release, usableCapacityInTiB and onboardingValidationKey
+    10. Verify that version of Prometheus is 4.10.0
```
Comment (jilju) [inline ocs_ci/helpers/managed_services.py:421]:
```suggestion
    10. Verify the version of Prometheus
```

File: ocs_ci/helpers/managed_services.py [hunk 3]
```diff
@@ -525,6 +526,14 @@ def verify_faas_provider_resources():
     log.info("Verifying managedFusionOffering onboardingValidationKey")
     assert len(offering_info["spec"]["config"]["onboardingValidationKey"]) > 700
 
+    # Verify that version of Prometheus is 4.10.0
```
Comment (jilju) [inline ocs_ci/helpers/managed_services.py:529]:
```suggestion
    # Verify the version of Prometheus
```

File: ocs_ci/helpers/managed_services.py [hunk 4]
```diff
@@ -525,6 +526,14 @@ def verify_faas_provider_resources():
...
+    prometheus_csv = csv.get_csvs_start_with_prefix(
+        constants.OSE_PROMETHEUS_OPERATOR, config.ENV_DATA["cluster_namespace"]
+    )
+    prometheus_version = prometheus_csv[0]["spec"]["version"]
+    log.info(f"Verifying Prometheus version: {prometheus_version}")
+    assert prometheus_version == config.ENV_DATA["prometheus_version"]
```
Comment (jilju) [inline ocs_ci/helpers/managed_services.py:535]: Please add an error message.

File: ocs_ci/helpers/managed_services.py (hunk 2 above)
Comment (ebondare) [reply ocs_ci/helpers/managed_services.py]: Thank you, fixed

File: ocs_ci/helpers/managed_services.py (hunk 4 above)
Comment (ebondare) [reply ocs_ci/helpers/managed_services.py]: Fixed, thank you

**Review by jilju — APPROVED** [review]

**Review by fbalak — APPROVED** [review]

**Pull request merged**